api/HTMLMediaElement.json

```
@@ -1647,23 +1647,35 @@
             "ie": {
               "version_added": "9"
             },
-            "opera": {
-              "version_added": true
-            },
-            "opera_android": {
-              "version_added": null
-            },
+            "opera": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "≤12.1",
+                "version_removed": "15"
+              }
+            ],
+            "opera_android": [
+              {
+                "version_added": "19"
+              },
+              {
+                "version_added": "≤12.1",
+                "version_removed": "14"
+              }
+            ],
             "safari": {
-              "version_added": null
+              "version_added": "9"
```

```suggestion
              "version_added": "4"
```

Based on WebKit trunk version 528.4:
https://trac.webkit.org/changeset/36893/webkit
https://trac.webkit.org/browser/webkit/trunk/WebCore/Configurations/Version.xcconfig?rev=36893